storage/object/reference-content/ssl-tls-certificate.mdx

```diff
@@ -113,14 +120,43 @@ cat private_key.pem cert.pem root_cert.pem > cert_chain.pem
 
 </Message>
 
-## Uploading your certificate
+### Tips for creating a certificate
+
+In general, SSL/TLS certificates can either be **self-signed** (signed by the subject of the certificate, e.g. the owner of the domain) or **CA-signed** (signed by a third party **C**ertificate **A**uthority which is publicly trusted). 
+
+**Self-signed certificates cannot be used with Edge Services, all certificates must be signed by a CA.**
```

Comment:
@RoRoJ just a precision here, it's not enough that the certificate is signed by a CA for us not to consider it's not self signed. The CA that signed the certificate needs to be known and trusted by Edge Services, and this won't be the case if this is the CA of a private company not in the business of SSL or domains. We also had a case where we didn't trust the Zero SSL company by default (which we should) and we then had to do a fix. Maybe it's worth mentioning that if the customers encounter self signed certificate errors in Edge Service and they believe that their cert is legit (signed by an official CA) then they should contact support.